src/toil_scripts/exome_variant_pipeline/exome_variant_pipeline.py

```diff
@@ -108,10 +108,10 @@ def preprocessing_declaration(job, config):
         job.fileStore.logToMaster('Ran preprocessing: ' + config.uuid)
         disk = '1G' if config.ci_test else '20G'
```

Comment:
Not a change here, but I've seen OOD failures with the preprocessing pipeline in the wild. I'll create a note for you me and @jvivian to go through this and audit the disk requirements before merging.